packages/components/src/drop-zone/index.js

```diff
@@ -118,23 +118,26 @@ export default function DropZoneComponent( {
 
 	if ( isDraggingOverElement ) {
 		children = (
-			<motion.div
+			<motion.span
 				variants={ backdrop }
 				initial={ disableMotion ? 'show' : 'hidden' }
 				animate="show"
 				exit={ disableMotion ? 'show' : 'exit' }
 				className="components-drop-zone__content"
 			>
-				<motion.div variants={ foreground }>
+				<motion.span
+					variants={ foreground }
+					className="components-drop-zone__content-foreground"
```

Comment:
The foreground is here to automatically _squeeze_ the content into a single line when the height is not big enough.